lib/OpenQA/WebAPI/Plugin/ObsRsync/templates/ObsRsync_logs.html.ep

```diff
@@ -0,0 +1,13 @@
+% layout 'bootstrap';
+% content_for 'head' => begin
+  %= asset 'codemirror.js'
+  %= asset 'codemirror.css'
+% end
+
+% title 'OBS synchronization Logs';
+<H2><%= 'OBS synchronization Logs: ' . $folder %></H2>
```

Comment:
I wanted to paste 'We use lower-case HTML tags everywhere. Please be consistent.' again but these files are also quite repetitive. Maybe just use one template for all these pages and stash the different variables on the controller side? I mean that's the purpose of templates, right?